### PR TITLE
7602 fix double econCommittee

### LINKED ISSUE #7602: econCommitteeCharter is started twice
## Describe the bug

As noted in https://github.com/Agoric/agoric-sdk/pull/7426#pullrequestreview-1388319867 , our main configuration results in two instances of `econCommitteeCharter` running:

```
      |-xsnap-worker,3475775 v16:zcf-b1-c1224-econCommitteeCharter
      |-xsnap-worker,3475783 v17:zcf-b1-c1224-econCommitteeCharter
```

This seems to not cause any critical problems in our `test-vaults-config`, but that's due to some fragile luck: this bit me recently bit me while working on #4548: in a test, governance actions failed because a smart wallet request used one instance to refer to it but the invitation that made it to the wallet was the other instance, so they didn't match. (Thanks @michaelfig for helping me diagnose that one!)

## To Reproduce
1. use `start-local-chain.sh` to start a node
2. `pstree --hide-threads`
3. see 2 different vats/processes with `econCommitteeCharter` in the name

## Expected behavior

Just 1 EC Charter.

## Platform Environment
 - what OS are you using? what version of Node.js?
 - is there anything special/unusual about your platform?
 - what version of the Agoric-SDK are you using? (run `git describe --tags --always`)

## Additional context

I thought this was a trivial config tweak, so I was tracking it under...

 - #7311

The two instances are a consequence of `startEconCharter` in two manifests that we use:

https://github.com/Agoric/agoric-sdk/blob/3fa74c5bb9675c4ed967cd9c7961ab4692a49cf7/packages/inter-protocol/src/proposals/startPSM.js#L425

https://github.com/Agoric/agoric-sdk/blob/3fa74c5bb9675c4ed967cd9c7961ab4692a49cf7/packages/inter-protocol/src/proposals/committee-proposal.js#L170

But when I made the trivial config tweak in 67420e8d1 to address the testing problem above, it caused some tests to fail. So this needs a bit of focused attention.

### PR DESCRIPTION
**Pull request by turadg**
closes: #7602 

## Description

Fixes the duplicate `econCommitteeCharter` start. In doing so moves boot-psm out of config and to only the two smart-wallet integration tests that depend on it.

While testing with a local chain, I encountered some of the smoketest scripts failing so this also has commits to fix those.

### Security Considerations

Non-duplicate governance is good.

### Scaling Considerations

--

### Documentation Considerations

--

### Testing Considerations

Ran `agops-governance-smoketest.sh` locally

### CODE REVIEW

File: packages/inter-protocol/src/proposals/price-feed-proposal.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ const sanitizePathSegment = name => {
 /**
  * Create inert brands (no mint or issuer) referred to by price oracles.
  *
- * @param {ChainBootstrapSpace} space
+ * @param {ChainBootstrapSpace & NamedVatPowers} space
```
Comment (dckc) [inline packages/inter-protocol/src/proposals/price-feed-proposal.js:36]: `NamedVatPowers` is landing too? that's a bit of a surprise...

**Review by dckc — APPROVED** [review]
Thanks a bunch!

I think `NamedVatPowers` doesn't belong and will fail lint. otherwise, looks good.

File: packages/inter-protocol/src/proposals/price-feed-proposal.js (hunk 1 above)
Comment (turadg) [reply packages/inter-protocol/src/proposals/price-feed-proposal.js:36]: the file is ts-nocheck. I left it in as documentation

File: packages/agoric-cli/src/lib/wallet.js [hunk 2]
```diff
@@ -106,7 +112,7 @@ export const coalesceWalletState = async (follower, invitationBrand) => {
  * }} opts
  */
 export const sendAction = async (bridgeAction, opts) => {
-  const offerBody = JSON.stringify(marshaller.toCapData(bridgeAction));
+  const offerBody = JSON.stringify(marshaller.toCapData(harden(bridgeAction)));
```
Comment (dckc) [inline packages/agoric-cli/src/lib/wallet.js]: I think @samsiegart just put this in another PR. We'll see which one lands 1st :)

File: packages/agoric-cli/test/agops-governance-smoketest.sh [hunk 3]
```diff
@@ -20,35 +20,23 @@ set -x
...
 
 # Accept invitation to economic committee
-COMMITTEE_OFFER=$(mktemp -t agops.XXX)
-bin/agops ec committee >|"$COMMITTEE_OFFER"
-jq ".body | fromjson" <"$COMMITTEE_OFFER"
-agoric wallet send --offer "$COMMITTEE_OFFER" --from gov1 --keyring-backend="test"
-# verify the offerId is readable from chain history
-agoric wallet show --from gov1
-COMMITTEE_OFFER_ID=$(jq -r ".body | fromjson | .offer.id" <"$COMMITTEE_OFFER")
+bin/agops ec committee --send-from gov1
```
Comment (dckc) [inline packages/agoric-cli/test/agops-governance-smoketest.sh:23]: :clap:

File: packages/agoric-cli/test/agops-governance-smoketest.sh [hunk 4]
```diff
@@ -20,35 +20,23 @@ set -x
...
-CHARTER_OFFER_ID=$(jq -r ".body | fromjson | .offer.id" <"$CHARTER_OFFER")
+bin/agops ec charter --send-from gov1
 
 ### Now we have the continuing invitationMakers saved in the wallet
 
 # Use invitation result, with continuing invitationMakers to propose a vote
 PROPOSAL_OFFER=$(mktemp -t agops.XXX)
 bin/agops psm proposePauseOffers --substring wantMinted --charterAcceptOfferId "$CHARTER_OFFER_ID" >|"$PROPOSAL_OFFER"
-jq ".body | fromjson" <"$PROPOSAL_OFFER"
+agoric wallet print --file "$PROPOSAL_OFFER"
```
Comment (dckc) [inline packages/agoric-cli/test/agops-governance-smoketest.sh:33]: :clap:

File: packages/inter-protocol/scripts/add-collateral-core.js [hunk 5]
```diff
@@ -62,64 +59,6 @@ export const defaultProposalBuilder = async (
   });
 };
 
-export const psmGovernanceBuilder = async ({
```
Comment (dckc) [inline packages/inter-protocol/scripts/add-collateral-core.js:65]: thanks for tidying this up!

File: packages/vats/decentral-demo-config.json [hunk 6]
```diff
@@ -151,4 +146,4 @@
     }
   },
   "defaultManagerType": "xs-worker"
-}
\ No newline at end of file
+}
```
Comment (dckc) [inline packages/vats/decentral-demo-config.json]: is that adding or removing the trailing newline?

**Review by dckc — APPROVED** [review]
lots of good clean-up in there

I'm not certain all of that is dead code, but I'm willing to risk it and to help deal with problems that may arise.

**Pull request merged**